### PR TITLE
Updates to the RHSSO perf test case

### PR DESCRIPTION
**Pull request by trepel**
# Description

Updates to the RHSSO performance testing test case template.

To verify:
- read through it
- you can try to create the jks file, if the steps are simple enough
- check the links if all of them are accessible
- not sure what else

### CODE REVIEW

File: test-cases/tests/performance/k03-run-performance-test-against-rhsso.md [hunk 1]
```diff
@@ -32,17 +46,32 @@ You should have an AWS IAM account in [QE-AWS account](https://068334777414.sign
...
    - **NOTE:** `-Djackson.version=2.10.1` is a workaround for RHSSO 7.4 only. Please remove this param from the test case when we upgrade to RHSSO 7.5
-9. Run
+
+9. Execute single performance test
+
+   `mvn verify -Ptest -Ddataset=1r_10c_100u`
+
+   > This is just to verify that everything is ok before running the actuall stress test. If the command passes ok you can proceed with the next step.
+
+   > If experiencing 404 not found issues, double check the RHSSO URL, make sure the `/auth` is appended. It might also be required to set Java Trust Store using `-DtrustStore=PATH_TO/TrustStore.jks -DtrustStorePassword=<password-if-it-is-password-protected>`.
```
Comment (STEPHINRACHEL) [inline test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:72]: May I know from where can I get this .jks file

Comment (trepel) [reply test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:72]: Ah, it is apparently not clear enough. There's new subsection called "How to create Trust Store JKS file"  withing "Prerequisities" section describing how to create that JKS file

Comment (trepel) [reply test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:72]: I guess I need to add something like `See "How to create Trust Store JKS file" section for details on how to create the JKS file.`

Comment (STEPHINRACHEL) [reply test-cases/tests/performance/k03-run-performance-test-against-rhsso.md:72]: Ah, nvm. my bad reading habit, I did not see the subsection under pre-requisite, I think that's enough.

**Pull request merged**